cpp/src/rolling/rolling.cu

```diff
@@ -76,4 +80,18 @@ std::unique_ptr<column> rolling_window(column_view const& input,
     input, preceding_window, following_window, min_periods, agg, stream, mr);
 }
 
+std::pair<std::unique_ptr<column>, std::unique_ptr<column>> windows_from_offset(
```

Comment:
Lets move this to the `window_utils.cu` with its detail counterpart.
And move `window_utils.cu` to `src/rolling`

Reply:
Done.